Gemfile

```diff
@@ -47,8 +47,8 @@ gem 'hypdf', '~> 1.0.10'          # PDFInfoAgent
 gem 'weibo_2', github: 'albertsun/weibo_2', branch: 'master'
 
 # GoogleCalendarPublishAgent and GoogleTranslateAgent
-gem 'google-api-client', '~> 0.13'
-gem 'google-cloud-translate', '~> 1.0.0', require: 'google/cloud/translate'
+gem 'google-api-client', '~> 0.34'
```

Comment:
Wow, we are so overdue... 🙇

Reply:
It doesn't look like Gemfile.lock is updated in sync.

Reply:
Yeah I'm really not a fan of how Ruby handles dependencies if I'm completely honest...
I should some free time on Wednesday to look into the test issue and this.